src/include/processor/mapper/plan_mapper.h

```diff
@@ -23,7 +23,8 @@ class PlanMapper {
         : storageManager{storageManager}, memoryManager{memoryManager},
           expressionMapper{}, catalog{catalog}, physicalOperatorID{0} {}
 
-    unique_ptr<PhysicalPlan> mapLogicalPlanToPhysical(LogicalPlan* logicalPlan);
+    unique_ptr<PhysicalPlan> mapLogicalPlanToPhysical(LogicalPlan* logicalPlan,
+        const expression_vector& expressionsToCollect, bool isDDLOrCopyCSV);
```

Comment:
I would suggest pass in `StatementType statementType` instead of `bool isDDLOrCopyCSV` here.

Reply:
Giving statementType to physicalPlan doesn't seem correct either. I will remove this field soon anyway.